packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3775,6 +3790,44 @@ export default class Meeting extends StatelessWebexPlugin {
     }
   }
 
+  /**
+   * Callback called when a relay event is received from meeting LLM Connection
+   * @param {any} e Event object coming from LLM Connection
+   * @private
+   * @returns {void}
+   */
+  private processRelayEvent = (e: any): void => {
```

Comment:
We can do better than `any` here

In the `reactions.types.ts`:

```ts

export type RelayEvent = {
  data: RelayEventData
};

type RelayEventData = {
  relayType: typeof REACTION_RELAY_TYPES['REACTION']
  reaction: Reaction;
  sender: Sender;
}

```

The good thing with this pattern, you can extend it later like this:

```ts

type RelayEventData =
  | {
      relayType: typeof REACTION_RELAY_TYPES['REACTION'];
      reaction: Reaction;
      sender: Sender;
    }
  | {
      relayType: typeof REACTION_RELAY_TYPES['TYPE_2'];
      something: boolean;
    };
```
TS understands the link between these type variants. It called Discriminated Unions, mode details here:
https://www.typescriptlang.org/docs/handbook/2/narrowing.html#discriminated-unions

Reply:
wow, thats a good read, thanks @maxinteger

Reply:
thanks for the suggestion!
Addressed